lib/backup/notifier/ses.rb

```diff
@@ -96,7 +106,15 @@ def notify!(status)
           }
         end
 
-        client.send_raw_email(email)
+        send_opts = {}
+        send_opts[:raw_message] = {}
+        send_opts[:raw_message][:data] = email.to_s
```

Comment:
I had purposely kept it like this, as it's copy/paste from `aws-sdk-rails` (https://github.com/aws/aws-sdk-rails/blob/9c1487dc58a0d42b5587f4749cbc10442d6aa023/lib/aws/rails/mailer.rb#L26-L28), but happy to change.